src/components/Popups/index.tsx

```diff
@@ -14,6 +15,8 @@ const MobilePopupWrapper = styled.div`
   max-width: 100%;
   margin: 0 auto;
   display: none;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
noticed the mobile toasts were touching the edge, fixed it:

<img width="544" alt="CleanShot 2023-09-05 at 09 54 54@2x" src="https://github.com/Uniswap/interface/assets/12100/bdb29b08-d61d-4be9-acc5-777f28b98147">